pkg/api/unversioned/register.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package unversioned
+
+func (*APIVersions) IsAnAPIObject()     {}
```

Comment:
@deads2k So there are some uses of `runtime.Object`s that will expect to be able to find a TypeMeta. However, they shouldn't crash if they don't find it, and they shouldn't end up operating on these particular types.

Later we can reevaluate this, but at least for the moment we don't want these types to have the apiVersion/kind fields, but they *are* (unversioned) api types.